automation/run-tests.sh

```diff
@@ -259,6 +259,15 @@ function run_tests {
             not test_linux_bridge_over_bond_over_port_in_one_transaction and \
             not test_explicitly_ignore_a_bridge_port' \
             ${nmstate_pytest_extra_args}"
+        exec_cmd "
+          env  \
+          PYTHONPATH=$CONTAINER_WORKSPACE/rust/src/python \
+          pytest \
+            $PYTEST_OPTIONS \
+            tests/integration/ethtool_test.py \
+            -k '\
+            not test_ethtool_invalid_feature' \
```

Comment:
Maybe we should change this test case to expect `NmstateValueError` as we should not allowing arbitrary feture string. The failure should happen at validation stage.

The `#[serde(deny_unknown_fields)]` can do the magic.

Reply:
Done, thanks!